src/server/game/Spells/SpellInfo.cpp

```diff
@@ -1675,7 +1675,7 @@ bool SpellInfo::IsChanneled() const
 
 bool SpellInfo::IsMoveAllowedChannel() const
 {
-    return IsChanneled() && HasAttribute(SPELL_ATTR5_CAN_CHANNEL_WHEN_MOVING);
+    return IsChanneled() && (HasAttribute(SPELL_ATTR5_CAN_CHANNEL_WHEN_MOVING) || !HasChannelInterruptFlag(AURA_INTERRUPT_FLAG_MOVE | AURA_INTERRUPT_FLAG_TURNING));
```

Comment:
Do i have to cast to ChannelInterruptFlag here?
Or do i have to add HasChannelInterruptFlag which accepts AuraInterruptFlags ?

See original commit

Reply:
Just access ChannelInterruptFlags[0] directly